pkgs/top-level/all-packages.nix

```
@@ -24674,11 +24674,11 @@ with pkgs;
 
   avrdudess = callPackage ../applications/misc/avrdudess { };
 
-  awesome-4-0 = callPackage ../applications/window-managers/awesome {
+  awesome_4_0 = callPackage ../applications/window-managers/awesome {
```

```suggestion
  awesome = callPackage ../applications/window-managers/awesome {
```

From what I see in the commit history, `awesome_4_0` is just historical baggage that hasn't been cleaned up.
As there are no other versions of `awesome` we might as well straight up change it to `awesome`.